zuul-core/src/main/java/com/netflix/zuul/niws/RequestAttempt.java

```diff
@@ -24,9 +24,9 @@
 import com.netflix.appinfo.InstanceInfo;
```

Comment:
I may be mistaken, but I thin this comes from discovery too.

Reply:
this comes from a different package, ie, `eureka-client` and not one of the ribbon deps. But yes, we want to move away from it, at some point, ideally.